apps/api/src/app/environments-v1/usecases/output-renderers/render-email-output.usecase.ts

```diff
@@ -50,7 +56,19 @@ export class RenderEmailOutputUsecase {
   }
 
   @Instrument()
-  private transformForAndShowLogic(body: string, fullPayloadForRender: FullPayloadForRender) {
+  private transformMailyDynamicBlocks(body: string, fullPayloadForRender: FullPayloadForRender) {
     return this.expandEmailEditorSchemaUseCase.execute({ emailEditorJson: body, fullPayloadForRender });
   }
 }
+
+export const stringifyDataStructureWithSingleQuotes = (value: unknown, spaces: number = 0): string => {
+  if (Array.isArray(value) || (typeof value === 'object' && value !== null)) {
+    const valueStringified = JSON.stringify(value, null, spaces);
+    const valueSingleQuotes = valueStringified.replace(/"/g, "'");
+    const valueEscapedNewLines = valueSingleQuotes.replace(/\n/g, '\\n');
+
+    return valueEscapedNewLines;
+  } else {
```

Comment:
`else` is not required.

```suggestion
  }
```